servers/src/main/java/tachyon/master/file/meta/InodeDirectory.java

```diff
@@ -177,9 +177,36 @@ public String toString() {
     return sb.toString();
   }
 
+  /**
+   * Converts the entry to an {@code InodeDirectory}.
```

Comment:
The convention I saw @HorizonNet apply is described here: https://github.com/amplab/tachyon/pull/2102